src/utils/data/provinces.ts

```diff
@@ -0,0 +1,26 @@
+import { get, update } from 'utils/api';
+
+const ranges = [
+  ['A', 2],
+  ['F', 33],
+];
+
+const sheetRange = `Provinces!${
```

Comment:
this sounds pretty magical lol. i would rather go for notion api, looks less fragile.

or you can leverage this instead https://www.npmjs.com/package/google-spreadsheet

as an optimization i suggest you to used the row metadata property to store the uuid of your record. it is invisible to google spreadsheet so people cannot tamper the data, and it's more optimized when you want to fetch one row by id.